src/components/CalendarDay.jsx

```diff
@@ -48,6 +55,9 @@ export default class CalendarDay extends React.Component {
         onClick={e => this.onDayClick(day, e)}
       >
         <span className="CalendarDay__day">{day.format('D')}</span>
+        {details && (
+          <span className="CalendarDay__details">{details}</span>
```

Comment:
I see. Because details defaults to a noOp which is truthy. I guess we could just set `const details = renderDetails(day);` and get rid of the `maybeRenderDetails` function but leave this as is.